docs/Installation.md

```diff
@@ -44,11 +44,11 @@ In your IEM open the Databus and launch the configurator.
 Add a user with this topic:
 `"ie/#"`
 
-![ie_databus_user](graphics/IE_Databus_User.PNG)
+![databususer](graphics/databususer.PNG)
```

Comment:
Please use a picture where the topic name is seen to avoid confusion.

Reply:
Hi Ahmed, I changed the picture of Databus